main.css

```diff
@@ -112,10 +118,12 @@ h1.headline {
 
 h2.headline {
   font-size: 2.1rem;
+  text-shadow: 2px 2px 5px #222;
```

Comment:
This shadow is pretty repetitious. There are several ways of handling this and perhaps you've considered all of them:

1. Do what you did and repeat the shadow
2. Create a compound selector like `.headline, .pageHeader__primary { }`
3. Create a new class and apply it to the HTML
4. Others

I think what you've done is fine, I'm just trying to make you aware of other possibilities. Once you start learning Sass, you'll see other ways to reduce this duplication through things like variables and mixins.